src/pip/_internal/operations/check.py

```diff
@@ -68,8 +68,8 @@ def check_package_set(package_set, should_ignore=None):
         def should_ignore(name):
             return False
 
-    missing = dict()
-    conflicting = dict()
+    missing = {}
+    conflicting = {}
```

Comment:
Unrelated change?

Reply:
This was caught via `check-builtin-literals`.